tests/roles/stop_openstack_services/tasks/main.yaml

```diff
@@ -1,46 +1,56 @@
 - name: set shell vars for stopping openstack services
   no_log: "{{ use_no_log }}"
   ansible.builtin.set_fact:
+  # TODO: check, this could actually work:
     stop_openstack_services_shell_vars: |
       CONTROLLER1_SSH="{{ controller1_ssh }}"
       CONTROLLER2_SSH="{{ controller2_ssh }}"
       CONTROLLER3_SSH="{{ controller3_ssh }}"
 
+- name: Set list of services to stop
+  set_fact:
+    services_to_stop:
+    - "tripleo_horizon.service"
+    - "tripleo_keystone.service"
+    - "tripleo_cinder_api.service"
+    - "tripleo_cinder_api_cron.service"
+    - "tripleo_cinder_scheduler.service"
+    - "tripleo_cinder_volume.service"
+    - "tripleo_cinder_backup.service"
+    - "tripleo_glance_api.service"
+    - "tripleo_manila_api.service"
+    - "tripleo_manila_api_cron.service"
+    - "tripleo_manila_scheduler.service"
+    - "tripleo_neutron_api.service"
+    - "tripleo_nova_api.service"
+    - "tripleo_nova_api_cron.service"
+    - "tripleo_nova_conductor.service"
+    - "tripleo_nova_metadata.service"
+    - "tripleo_nova_scheduler.service"
+    - "tripleo_nova_vnc_proxy.service"
+    - "tripleo_ovn_cluster_northd.service"
+    - "tripleo_placement_api.service"
+
+- name: Set list of Pacemaker resources to stop
+  set_fact:
+    pacemaker_resources_to_stop:
+    - "openstack-cinder-volume"
+    - "openstack-cinder-backup"
+    - "openstack-manila-share"
+
 - name: stop control plane services
   no_log: "{{ use_no_log }}"
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
     {{ stop_openstack_services_shell_vars }}
 
-    ServicesToStop=("tripleo_horizon.service"
-                    "tripleo_keystone.service"
-                    "tripleo_cinder_api.service"
-                    "tripleo_cinder_api_cron.service"
-                    "tripleo_cinder_scheduler.service"
-                    "tripleo_cinder_volume.service"
-                    "tripleo_cinder_backup.service"
-                    "tripleo_glance_api.service"
-                    "tripleo_manila_api.service"
-                    "tripleo_manila_api_cron.service"
-                    "tripleo_manila_scheduler.service"
-                    "tripleo_neutron_api.service"
-                    "tripleo_nova_api.service"
-                    "tripleo_nova_api_cron.service"
-                    "tripleo_nova_api.service"
-                    "tripleo_nova_conductor.service"
-                    "tripleo_nova_metadata.service"
-                    "tripleo_nova_scheduler.service"
-                    "tripleo_nova_vnc_proxy.service"
-                    "tripleo_ovn_cluster_northd.service"
-                    "tripleo_placement_api.service")
+    ServicesToStop=({{ services_to_stop | join(' ') }})
```

Comment:
This should use the shell vars as it was, the tests have to run what's written in the documentation. We're not aiming for "prettiest Ansible", we're aiming for a test suite that verifies what's written in the docs and will be executed by admins, which often means having to use shell constructs instead of Ansible constructs in the test suite.